### ISSUE TITLE
ReferenceError: response is not defined

### ISSUE THREAD

**Issue by vladholubiev** [doapp-ryanp/dynamodb-local#26]
Today our tests started failing in CircleCI. No code changes were made.

Is this related to the recent release https://aws.amazon.com/about-aws/whats-new/2019/02/amazon-dynamodb-local-adds-support-for-transactional-apis-on-demand-capacity-mode-and-20-gsis/ ?

```
yarn run v1.12.3
$ export ENVIRONMENT=local && jest src --runInBand
/home/circleci/repo/node_modules/dynamodb-local/index.js:164
                        response.headers['location'] + ': ' + redirectResponse.statusCode));
                        ^

ReferenceError: response is not defined
    at ClientRequest.<anonymous> (/home/circleci/repo/node_modules/dynamodb-local/index.js:164:25)
    at Object.onceWrapper (events.js:315:30)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:558:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:119:17)
    at TLSSocket.socketOnData (_http_client.js:454:20)
    at emitOne (events.js:116:13)
    at TLSSocket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Exited with code 1
```

**Comment by yesteph** [comment]
It seems the default download link references US-WEST region..
But the link is KO, even on the AWS [documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)

**Comment by antogyn** [comment]
The link is KO, and on this line : https://github.com/rynop/dynamodb-local/blob/cc53bb2cdad344475fc3f1448d4c76b8cf938302/index.js#L164
`response` should be `redirectResponse`

For now I worked around it by changing the download url :

```js
DynamoDbLocal.configureInstaller({
  downloadUrl: 'https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/dynamodb_local_latest.tar.gz',
});
```

**Comment by rynop** [comment]
When you say "KO" are you saying you are getting a 404 on https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz ?

As of 10:15a central this URL is working for me (without returning a redirect or anything).

My guess is they rolled out new features as you mention @vladgolubev and temporarily broke the URL.

@antogyn thanks for pointing out that bug, I've [fixed](https://github.com/rynop/dynamodb-local/commit/c52607e85f3605123c6484339e1e4c99c2d0d7d2) and pushed a new minor ver to NPM.

Let me know if all is well and I'll close this issue.  Thanks all.

**Comment by vladholubiev** [comment]
I can confirm as well, everything is working again, thanks @rynop!

Closed by c52607e85f3605123c6484339e1e4c99c2d0d7d2
